src/main/java/at/hannibal2/skyhanni/features/garden/GardenPlotAPI.kt

```diff
@@ -33,6 +33,10 @@ object GardenPlotAPI {
         "spray.target",
         "§a§lSPRAYONATOR! §r§7You sprayed §r§aPlot §r§7- §r§b(?<plot>.*) §r§7with §r§a(?<spray>.*)§r§7!"
     )
+    private val portableWasherPattern by patternGroup.pattern(
+        "spray.cleared",
```

Comment:
in the future there could be other means to clear a spray. use a more precise pattern key. e.g. spray.cleared.portablewasher